frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -225,6 +248,48 @@ export class EnrolledItemCard extends React.Component<
     )
   }
 
+  renderVerifiedUnenrollmentModal(enrollment: RunEnrollment) {
+    const { verifiedUnenrollmentModalVisibility } = this.state
+    let isOpen = false
+    if (verifiedUnenrollmentModalVisibility[enrollment.id] !== undefined) {
+      isOpen = verifiedUnenrollmentModalVisibility[enrollment.id]
+    }
+
+    return (
+      <Modal
+        id={`verified-unenrollment-${enrollment.id}-modal`}
+        className="text-center"
+        isOpen={isOpen}
+        toggle={() =>
+          this.toggleVerifiedUnenrollmentModalVisibility(enrollment.id)
+        }
+      >
+        <ModalHeader
+          toggle={() =>
+            this.toggleVerifiedUnenrollmentModalVisibility(enrollment.id)
+          }
+        >
+          Unenroll From {enrollment.run.course_number}
+        </ModalHeader>
+        <ModalBody>
+          <p>
+            You are enrolled in the certificate course for{" "}
+            {enrollment.run.course_number} {enrollment.run.title}. You can't
+            unenroll from this course from My Courses.
```

Comment:
is this clear?

```suggestion
            You are enrolled in the certificate track for{" "}
            {enrollment.run.course_number} {enrollment.run.title}. You can't
            unenroll from this course from My Courses.
```

Reply:
Updated this in https://github.com/mitodl/mitxonline/pull/602/commits/911a07af40220831f141e14b33813ccd6e560e55 .